pageserver/src/consumption_metrics/metrics.rs

```diff
@@ -287,16 +269,9 @@ where
                     continue;
                 }
             }
-
-            tenant_resident_size += timeline.resident_physical_size();
         }
 
-        if timelines_len == 0 {
-            // Force set it to 1 byte to avoid not being reported -- all timelines are offloaded.
-            tenant_resident_size = 1;
```

Comment:
I thought the billing team is still using this metrics? The removed code contains a bug fix a week ago. cc @Daniel-ef

Reply:
@skyzh The description for https://github.com/neondatabase/neon/pull/11648 talks about setting synthetic size to 1, but the code sets resident size to 1 (and then this PR removes resident size code).  Is there something subtle going on, or was the intent of 11648 to set synthetic size to 1 rather than resident size?

Reply:
Ok I might have set the wrong size in that patch 😅

Reply:
new fix: https://github.com/neondatabase/neon/pull/11731